reconcile/gql_definitions/vault_instances/vault_instances.gql

```diff
@@ -0,0 +1,58 @@
+# qenerate: plugin=pydantic_v1
+
+query VaultInstances {
+  vault_instances: vault_instances_v1 {
+    name
+    address
+    auth {
+      provider
+      secretEngine
+      ... on VaultInstanceAuthApprole_v1 {
+      roleID {
+        path
+        field
```

Comment:
Any reason to not use fragment `... VaultSecret` here? It has 2 additional fields `version` and `format`, which are redundant in your case, but you can neglect them. The extra "gql query overhead" wouldn't hurt and you'd greatly reduce the number of generated classes.

Reply:
(Same applies to all other VaultSecret occurrences in this query)

Reply:
Changed all the occurrences of the VaultSecret fragment in all the queries